pss/pss.go

```diff
@@ -740,6 +741,7 @@ func sendMsg(p *Pss, sp *network.Peer, msg *message.Message) bool {
 //// successfully forwarded to at least one peer.
 func (p *Pss) forward(msg *message.Message) error {
 	metrics.GetOrRegisterCounter("pss.forward", nil).Inc(1)
+	defer metrics.GetOrRegisterResettingTimer("pss.forward.time", nil).UpdateSince(time.Now())
```

Comment:
Remove the counter from the line above please, as there is already a counter part of the timer.